.github/default.gitignore

```diff
@@ -0,0 +1,224 @@
+# General Files
+*.log
+*.tmp
+*.bak
+*.swp
+*.swo
+.DS_Store
+Thumbs.db
+Desktop.ini
+._*
+.Trashes
+.Spotlight-V100
+.TemporaryItems
+.Trash-*
+*.pid
+*.pid.lock
+*.seed
+*.db
+*.dbmdl
+*.tlog
+*.tmproj
+*.tmp_proj
+*.un~
+*.user
+*.userosscache
+*.userprefs
+*.vbproj.user
+*.vssscc
+*.vspscc
+*.suo
+*.cache
+*.csproj.user
+*.sln.docstates
+*.iws
+
+# Byte-compiled / Optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+*.so
+
+# Distribution / Packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+
+# Virtual Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# IDE / Editor Configurations
+/.idea/
+*.iml
+/.vscode/
+*.code-workspace
+*.sublime-project
+*.sublime-workspace
+/.vs/
+/.atom/
+/.eclipse/
+/.netbeans/
+/.intellij/
+/.jbuilder/
+/.rubymine/
+/.mine/
+/.textmate/
+/.vim/
+/.emacs.d/
+*.sw*
+
+# Security - Sensitive files
+*.pem
+*.key
+*.crt
+*.csr
+*.gpg
+*.asc
+.secret
+*.vault
+*.token
+*.apikey
+*.credentials
+secrets.yml
+credentials.yml
+config/dev.yml
+config/private.yml
+
+# Dependency Directories
+/node_modules/
+/bower_components/
+/.jekyll-cache/
+/vendor/
+
+# Go specific
+/bin/
+/pkg/
+*.o
+*.a
+*.out
+**/vendor/
+Gopkg.lock
+Gopkg.toml
+go.sum
+go.mod
+
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Ruby specific
+/.bundle/
+/.yardoc
+/_yardoc/
+/coverage/
+/doc/
+/pkg/
+/spec/reports/
+/tmp/
+*.gem
+*.rbc
+.capistrano/
+.rspec
+.ruby-version
+.ruby-gemset
+# Ignore Byebug command history file.
+.byebug_history
+
+# Cloud Foundry specific
+manifest.yml
+manifest-*.yml
+*.env
```

Comment:
I think these will cause problems, for example:
[we have a `manifest.yml` here])https://github.com/cloud-gov/cf-hello-worlds/blob/8a17d850b6cf13e679754da0adfc0b62c5245fcb/go-hello/manifest.yml#L4)

Including manifests is a somewhat-common pattern, and there are definitely ways to do it safely